src/supermarket/config/locales/en.yml

```diff
@@ -38,7 +38,8 @@ en:
     undeprecated: "%{cookbook} is no longer deprecated."
     featured: "%{cookbook} is now %{state}."
     deprecate_with_deprecated_failure: "This cookbook cannot be deprecated in favor of a deprecated cookbook."
-    activity: "%{maintainer} released v%{version} of the %{cookbook} cookbook."
+    activity: "%{maintainer}: %{cookbook} v%{version} released by"
```

Comment:
Ugh. I'd forgotten that the text was handled by an I18n entry. "released by ..." and then the rest of the statement finished back in the view makes my brain itch.

🤔 

How awful would ...

👩‍🍳 Chef: rsyslog v5.1.0 released by Timothy Smith 👱 

... look? I think I'm OK with that. Curious what you think. That would let us keep a single I18n `activity` statement—it would need to take a new parameter like `publisher`—and only do the gravataration in the view.

Reply:
that makes sense to me; the awkwardness made me itch too. let me spin that up and we'll see how it looks.